karspexet/templates/base.html

```diff
@@ -0,0 +1,91 @@
+{% load menu_tags cms_tags sekizai_tags staticfiles%}
```

Comment:
What does this one do? I mean shouldn't it be _inside_ the html tag? Also a `<!DOCTYPE html>` would be nice pretty please.